From 7b1f102db70fd8c86f21c722669c9f7337367eee Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Sat, 8 Jan 2022 17:02:38 +0000
Subject: [PATCH 1/1] patched for sh

Exec Command issue with bad character for busybox sh.
This patches by removing bad character so that it can run with busybox

---
 src/targetPacks/assemble.targets | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/targetPacks/assemble.targets b/src/targetPacks/assemble.targets
index 6b884ce2..e5009793 100644
--- a/src/targetPacks/assemble.targets
+++ b/src/targetPacks/assemble.targets
@@ -13,7 +13,7 @@
     <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Assemble TargetingPacks src." />
     <MakeDir Directories="@(TargetingPackSrc->'$(ArtifactsTFMPackTemp)$(MSBuildProjectName)/%(RecursiveDir)')" />
     <!-- Note: Hack below to not fill up build logs.  Ilasm produces warning on validly disassembled il src.  The awk below eats just that warning -->
-    <Exec Command="set -o pipefail;$(IlasmToolPathSB)ilasm %(TargetingPackSrc.Identity) -dll -quiet -nologo -output=$(ArtifactsTFMPackTemp)$(MSBuildProjectName)/%(RecursiveDir)%(TargetingPackSrc.Filename).dll |&amp; awk '!/warning : Method has no body/'" IgnoreStandardErrorWarningFormat="true"/>
+    <Exec Command="set -o pipefail;$(IlasmToolPathSB)ilasm %(TargetingPackSrc.Identity) -dll -quiet -nologo -output=$(ArtifactsTFMPackTemp)$(MSBuildProjectName)/%(RecursiveDir)%(TargetingPackSrc.Filename).dll | awk '!/warning : Method has no body/'" IgnoreStandardErrorWarningFormat="true"/>
     <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Done assembling TargetPacks src." />
   </Target>
 </Project>
-- 
2.34.1

